app/src/api/dataSource/ClientApiDataSource.ts

```diff
@@ -153,6 +195,18 @@ export class ClientApiDataSource implements ClientApi {
       },
       config,
     );
+    // @ts-expect-error: Property 'inner' does not exist on type 'RpcError'
+    if (response.error?.inner?.response?.data === 'Token not valid.') {
+      await getNewJwtToken({ refreshToken, getNodeUrl });
```

Comment:
Check for status code of response, if its 403 and you have access then token try refreshing.